packages/components/src/tooltip/index.tsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import * as Ariakit from '@ariakit/react/tooltip';
+
+/**
+ * WordPress dependencies
+ */
+import { useInstanceId } from '@wordpress/compose';
+import { Children } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import type { TooltipProps } from './types';
+import Shortcut from '../shortcut';
+import { positionToPlacement } from '../popover/utils';
+
+/**
+ * Time over anchor to wait before showing tooltip
+ */
+export const TOOLTIP_DELAY = 700;
+
+function Tooltip( props: TooltipProps ) {
+	const { children, delay = TOOLTIP_DELAY, position, shortcut, text } = props;
+
+	const baseId = useInstanceId( Tooltip, 'tooltip' );
+	const describedById = text || shortcut ? baseId : undefined;
+
+	const isOnlyChild = Children.count( children ) === 1;
+	// console error if more than one child element is added
+	if ( ! isOnlyChild ) {
+		if ( 'development' === process.env.NODE_ENV ) {
+			// eslint-disable-next-line no-console
+			console.error(
+				'Tooltip should be called with only a single child element.'
+			);
+		}
+	}
+
+	const DEFAULT_PLACEMENT = 'bottom';
+
+	const tooltipStore = Ariakit.useTooltipStore( {
+		//@ts-ignore
```

Comment:
I opened a separate PR for @mirka's [suggested diff ](https://github.com/WordPress/gutenberg/pull/48440#discussion_r1312093852) to keep the history with Popover and so it doesn't get lost here. 

The ignore above will be removed once that PR is merged.